<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FGECreateCylinderMeshPropertiesForm</class>
 <widget class="QWidget" name="FGECreateCylinderMeshPropertiesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid ;
	border-color: rgb(156, 156, 156);
    margin-top: 10px;
    font-size: 14px;
    border-radius: 3px;
}

QGroupBox::title {
	top:0px;
	left : 15px;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px 2px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>New Cylinder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="name_object_lineEdit">
        <property name="text">
         <string>new cylinder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string> Base Radius </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="base_radius_doubleSpinBox">
        <property name="maximum">
         <double>999999.989999999990687</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> Top Radius </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="top_radius_doubleSpinBox">
        <property name="maximum">
         <double>999999.989999999990687</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Height </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="height_doubleSpinBox">
        <property name="maximum">
         <double>999999.989999999990687</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>name :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>sector</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="sector_spinBox">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>102</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="create_pushButton">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="top_mesh_groupBox">
     <property name="title">
      <string>Top Mesh :</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QSpinBox" name="top_stack_count_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string> Stack Count </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="midle_mesh_groupBox">
     <property name="title">
      <string>Midle Mesh :</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QSpinBox" name="midle_stack_count_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string> Stack Count </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="base_mesh_groupBox">
     <property name="title">
      <string>Base &amp;Mesh</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QSpinBox" name="base_tack_count_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string> Stack Count </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
